package.json

```diff
@@ -58,7 +58,7 @@
     "src"
   ],
   "dependencies": {
-    "immer": "^6.0.1",
+    "immer": "^7.0.0",
```

Comment:
Not that it would be *really* necessary, but want to make this a 7.0.1 just to be more explicit? (& update the package.lock, which will be necessary for the tests to pass) 
Other than that, I think it's ready to go :)

Reply:
yep, exactly

Reply:
updated!